src/bootstrap/flags.rs

```diff
@@ -259,8 +262,10 @@ Arguments:
             let maybe_rules_help = Builder::get_help(&build, subcommand.as_str());
             extra_help.push_str(maybe_rules_help.unwrap_or_default().as_str());
         } else {
-            extra_help.push_str(format!("Run `./x.py {} -h -v` to see a list of available paths.",
-                     subcommand).as_str());
+            if subcommand.as_str() != "clean" {
+                extra_help.push_str(format!("Run `./x.py {} -h -v` to see a list of available paths.",
```

Comment:
This line is too long. CI rejects lines with >100 chars. Perhaps you should change that `else { if ... { ... } }` into `else if ... { ... }`.

```
[00:03:16] tidy error: /checkout/src/bootstrap/flags.rs:266: line longer than 100 chars
```